### ISSUE TITLE
Password locator timeout

### ISSUE THREAD

**Issue by mmospanenko** [altryne/chatGPT-telegram-bot#47]
I use latest version docker

<img width="504" alt="image" src="https://user-images.githubusercontent.com/2766211/208917010-29cd42e4-2721-452a-a6a4-295c01ccffb3.png">

**Comment by altryne** [comment]
It feels like they are changing the interface too often, I entered the passwords manually tbh

**Comment by mmospanenko** [comment]
yes, they don't have 'password' field directly here :)
<img width="392" alt="image" src="https://user-images.githubusercontent.com/2766211/209068607-9d8a0472-ccd7-4bff-9adf-46cafc66c937.png">

I did'n use this service and can't say if there was captcha earlier

For remote server it is important to have ability to login you know. Any ideas how to workaround?

**Comment by mmospanenko** [comment]
yeah, main 'script' tied on selectors, very difficult to debug and maintain, I tried but don't have enough time, will be great to move them out and cover with tests for each part of data. For example, it couldn't get the last message now (list is empty), etc.

**Comment by trongkyKMA** [comment]
Same 
![image](https://user-images.githubusercontent.com/18124512/209564504-edd6786d-c077-4a7b-be48-4cbd5282d55f.png)

**Comment by Alvaspart** [comment]
same here. Looks like cloudflare waiting for captcha first then Login  button occurs
